boards/arm/nucleo_f411re/nucleo_f411re.dts

```diff
@@ -20,7 +20,7 @@
 
 &usart1 {
 	current-speed = <115200>;
-	pinctrl-0 = <&usart1_pins_a>;
+	pinctrl-0 = <&usart1_pins_b>;
```

Comment:
Why did you change the usart1 pins? Is there a conflict with I2C? If yes, you should also update board's doc and pinmux.c files.

Reply:
And ideally,you should split changes into 2 commits, one for I2C the other for uart and justify the change

Reply:
You are quite right. Thank must have been a mistake. Good catch.